### ISSUE TITLE
Table cell missing context on Features page

### ISSUE THREAD

**Issue by christhepianist** [samvera/hyku#3082]
vpat.hykudemo.org
Version 7.1

Descriptive summary
Table data cells are not connected to column or row headers:
<img width="930" height="1091" alt="Image" src="https://github.com/user-attachments/assets/ca2c5ac8-b7d8-4460-91f2-69d9d2017a86" />
<img width="536" height="208" alt="Image" src="https://github.com/user-attachments/assets/008c5c43-fa81-4f74-897b-962659da7d8c" />

Rationale
Accessibility issue remediation

Sighted users can make a visual association between table header and data cells. This context gives meaning to the data.

Screen reader users must rely on a programmatic relationship being made between header and data cells. As each data cell is read out, the relevant row or column header is also announced. In this way, the data has context.

It is best practice to limit table use to simple data sets. Complex tables are difficult to understand and there may be other, more accessible ways to present the content.

Help Center: https://help.siteimprove.com/support/solutions/articles/80001058790-accessibility-rule-table-cell-missing-context-explained
WCAG 2.2: https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships
Siteimprove Alfa: Technical documentation for rule SIA-R77: https://alfa.siteimprove.com/rules/sia-r77

Expected behavior
See above.

Actual behavior
See above.

Steps to reproduce the behavior

- Open the page
- Use the SiteImprove browser extension

Related
